test/test-applications/integrations/TestApplication.Http.NetFramework/Helpers/TcpPortProvider.cs

```diff
@@ -0,0 +1,42 @@
+// <copyright file="TcpPortProvider.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Net;
+using System.Net.Sockets;
+
+namespace TestApplication.Http.NetFramework.Helpers;
+
```

Comment:
It looks like a copy paste from test\IntegrationTests\Helpers\TcpPortProvider.cs.

Any chance to avoid it, by including this file into the compilation?

Reply:
I was thinking about @Kielek way at first as there is only 1 use case yet. No strong opinions as well.

Reply:
Remembered now that the copy reason was that when link referencing the source, it's going to be alienating namespace for test apps.

Reply:
Sure